data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -15,7 +15,7 @@
 
     [story]
         [part]
-            story= _ "After reaching the dwarvish keep and tending to their wounded, the humans led by Tallin and the dwarves led by Lord Hamel hurried deeper into the caverns for a hasty council."
+            story= _ "After reaching the dwarvish keep and tending to their wounded, the humans led by Tallin, and the dwarves led by Lord Hamel, hurried deeper into the caverns for a hasty council."
```

Comment:
I still slightly prefer less commas here. The commas add pauses, but the action is that they're hurrying into a hasty meeting.

Reply:
For me it's not about correctness or pauses but readability in this case. I feel like it's a bit hard to read with it entirely unbroken? Don't know if that helps you or not.

Reply:
Okay, that sways me.